src/main/java/org/eclipsefoundation/git/eca/resource/ValidationResource.java

```diff
@@ -56,6 +57,10 @@
 public class ValidationResource {
   private static final Logger LOGGER = LoggerFactory.getLogger(ValidationResource.class);
 
+  @Inject
+  @ConfigProperty(name = "eclipse.mail.whitelist")
```

Comment:
I think we should avoid the terms whitelist/blacklist when possible. The general trend is to use allowlist and denylist (or excludelist) instead which, IMO, convey exactly the same concepts.

Reply:
Good catch! I think I got all of the places I used the outdated terminology, thanks for pointing it out!